woodwork/statistics_utils/_get_box_plot_info_for_column.py

```diff
@@ -1,24 +1,65 @@
 import numpy as np
 import pandas as pd
 
+from woodwork.statistics_utils._determine_best_outlier_method import (
+    _determine_best_outlier_method,
+    _sample_for_medcouple,
+)
 from woodwork.utils import import_or_none
 
 dd = import_or_none("dask.dataframe")
 ps = import_or_none("pyspark.pandas")
 
 
+def _determine_coefficients(mc):
+    if mc >= 0:
+        lower_coef = -3.5
+        higher_coef = max(1, 4 * (1 - mc))
+    else:
+        lower_coef = min(-1, -4 * (1 + mc))
+        higher_coef = 3.5
+    return lower_coef, higher_coef
```

Comment:
One of the things that I noticed when going through the datasets that had skewed columns was that when a series had a large medcouple statistic, the corresponding upper or lower bound (depending on if the skew was right or left) was much larger than it should have been. This is due to the part of the formula where `e is raised to the power of (coefficient x medcouple stat)`.

This becomes an exponentially larger quantity which when multiplied by 1.5 and the IQR, can lead to bounds far outside of what might be helpful. The logic here works with that in mind and limits some of the larger quantities that result from a medcouple statistic closer to -1 and 1. An important reason for this is that a medcouple magnitude near 1 doesn't necessarily mean that the outliers are incredibly far away from the bulk of the observations, it also relates to the body of the distribution and the dispersion of the points.

Reply:
Interesting - so this will actually mean we see more outliers in medcouple by reducing the size of the whiskers? The constant coefficient is just a part of this statistic?

Reply:
Yes reducing the size of the whiskers means we'll see more outliers. In the end it depends on the mindset we want to have and this can even become a configurable parameter.

Do we want to err on the side of caution and make it smaller to avoid accidentally removing values that would have been outliers? Or do we want to more freely set outlier boundaries and if observations that would previously have been outliers end up flying under the radar and ignored, then so be it.